pulp_container/app/models.py

```diff
@@ -200,6 +200,50 @@ class Meta:
         unique_together = (("name", "tagged_manifest"),)
 
 
+class ImageManifestSignature(Content):
```

Comment:
my suggestion would be to rename thi class to a more generic `ManifestSignature`, i  believe `container/image` library does not distinguish between image and list.